src/QMCHamiltonians/ECPComponentBuilder.h

```diff
@@ -86,10 +86,12 @@ class ReadFileBuffer
   char *cbuffer;
   std::ifstream *fin;
   Communicate *myComm;
+  int get_file_length(std::ifstream *f);
```

Comment:
How to write get_file_length was mentioned in the bug report on OO vs. functional.   Yes, it could operate directly on fin and write to length.     However, I think that makes it harder to understand in isolation vs. using a more functional style.